README.md

```diff
@@ -45,68 +45,55 @@ imports: [
 ...
 ```
 
-## Documentation
+## 🍿 Documentation
 
-- Une documentation sur l'utilisation des composantes de la librairie avec des exemples de code est disponible [ici](https://services.interne.montreal.ca/bao-storybook). Le site de documentation est construit avec l'outil [Storybook](https://storybook.js.org/)
-- La documentation du _Ville de Montréal Design System_ est disponible [ici](https://zeroheight.com/575tugn0n/p/139208-ville-de-montral-design-system). Celle-ci explique quelles composantes utiliser selon le contexte.
+- Code examples is available [here](https://services.interne.montreal.ca/bao-storybook).
+- The design system documentation is available [here](https://zeroheight.com/575tugn0n/p/139208-ville-de-montral-design-system). This explains which components to use depending on the context.
 
 ## Versionning
 
-| Angular UI Version                  | Version d'Angular                      |
+| Angular UI Version                  | Angular Version                        |
 | ----------------------------------- | -------------------------------------- |
 | @villedemontreal/angular-ui@^13.0.0 | Angular 13. Compatible avec Angular 12 |
 | @villedemontreal/angular-ui@^14.0.0 | Angular 14                             |
 | @villedemontreal/angular-ui@^15.0.0 | Angular 15                             |
 
-## Philosophie de design
+📢 We don't use semver in releases since `major` is for Angular release to make it easier for you to understand and manage releases. We recommend using `~` and not `^` on versions such as `~15.3.0` (_MAJOR.MINOR.PATCH_) and not `^15.3.0`, because we reserve the right to make breaking changes ☝️ in components in `minor` releases. `~` to automatically update `patches`.
+
+## Design philosophy
 
-### De la composition plutôt que de la configuration
+### Composition rather than configuration
 
-Lorsque possible nous voulons laisser le développeur libre de composer le contenu de sa composante comme il le souhaite.
-Par contre, nous recommandons de suivre les recommandations du [sytème de design de la Ville de Montréal](https://zeroheight.com/575tugn0n/p/139208-ville-de-montral-design-system).
-Ceci veut dire qu'il est préférable de pouvoir injecter du contenu dans la composante via des balises `<ng-content></ng-content>` plutôt que d'avoir une liste interminable de `@Input` permettant d'avoir l'affichage/comportement désiré.
-L'idée générale est d'utiliser les `@Input()` pour tout ce qui est 'state' de la composante et les 'child-components' pour tout ce qui est contenu.
+When possible we want to leave the developer free to compose the content of his component as he wishes.
```

Comment:
... to compose the content of THEIR component as THEY wish (pronon non genré puisque les contributeurs ne sont pas que des hommes ;) )